core/src/main/resources/static/media/img/en_GB/app-store-badge.svg

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There is already an assert. However, my laptop is set to `en-GB` so Thymeleaf seems to ignore the `LocaleContextHolder.setLocale(Translator.EN_US_LOCALE);`